bundles/CoreBundle/Resources/translations/de.json

```diff
@@ -435,6 +435,7 @@
     "columns": "Spalten",
     "children_grid": "Kinder Tabelle",
     "only_children": "Nur direkte Kindelemente",
+    "only_unreferenced": "Nur nicht referenzierte",
```

Comment:
No need for de.json. en is the master file

Reply:
Is only a draft, please do not review yet. We try to teach also other colleagues to create PRs. We will fix it and convert it to a real PR soon.